### DISCUSSION TITLE
Is it possible to maintain the see-through effect even when tabbed off of VSCode?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [illixion/vscode-vibrancy-continued#55] [Q&A]
I noticed the see-through effect disappears after tabbing off of VSCode and returns when I re-focus.
![Screen Recording 2023-02-06 at 1 43 50 PM](https://user-images.githubusercontent.com/20440899/217112329-bcf21944-b0a8-4cc2-bbc9-ce308942771f.gif)

**Comment by illixion** [comment]
No solutions yet, this suggestion is tracked under #36.
